geometry_msgs/Point curr
geometry_msgs/Point dest
---
bool is_updated_map

